xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -225,9 +299,26 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Reads the next JSON token value from the source and parses it to an <see cref="T:System.Int32" />.</summary>
+        <returns>The UTF-8 encoded token value parsed to an <see cref="T:System.Int32" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `GetInt32` method throws an exception if the entire UTF-8 encoded token value cannnot be parsed to a <xref:System.Int32> value.
```

Comment:
It feels sort of weird to me to have a remark that says "this throws" and then also have the exceptions and causes table.

Reply:
you're right. I'll remove those.